hypothesis-python/src/hypothesis/strategies.py

```diff
@@ -309,6 +320,7 @@ def booleans():
 def floats(
     min_value=None, max_value=None, allow_nan=None, allow_infinity=None
 ):
+    # type: (Real, Real, bool, bool) -> SearchStrategy
```

Comment:
Why are we using this format here rather than adding comments to each argument like we do elsewhere? I find that significantly more readable with larger argument lists.

Reply:
My intuition was to prefer one line if the signature fits on a line, but happy to switch to the other style consistently.

Eg. document in our style guide "one-line style for one- or two-argument functions, for three+ arguments use a comment on each line".

Reply:
That seems like a good rule, yeah. I mainly want to make sure that we don't have to line up type signatures in our head, and it doesn't seem onerous to do that for 2-argument functions.